fusion_report/app.py

```diff
@@ -238,8 +239,8 @@ def score(self, params: Namespace) -> None:
 
             db_score: float = db_hits / db_provided
 
-            score: float = float("%0.3f" % (0.8 * tool_score + 0.2 * db_score))
-            score_explained = f"0.8 * ({len(fusion.tools)} / {tools_provided}) + 0.2 * ({(db_hits)} / {db_provided})"
+            score: float = float("%0.3f" % (0.5 * tool_score + 0.5 * db_score))
+            score_explained = f"0.5 * ({len(fusion.tools)} / {tools_provided}) + 0.5 * ({(db_hits)} / {db_provided})"
```

Comment:
This is not very explanatory really. I wonder if we should break it down and give a bit more details

Reply:
here or in the docs

Reply:
Tried to expand the documentation